newauth/views.py

```diff
@@ -14,6 +15,14 @@
 from newauth.api import login as auth_login, logout as auth_logout
 from newauth.forms import BasicAuthForm
 
+
+def _is_safe_url(redirect_to, request):
+    if django.VERSION < (1, 11):
+        return is_safe_url(redirect_to, host=request.get_host())
+    else:
+        return is_safe_url(redirect_to, allowed_hosts={request.get_host()})
```

Comment:
*note*: As this function is a private API [^1], there isn't any documents for this function. If you want to know the interface of this function, you can see the source code online [^2]

Copy of the function signature (django 2.1):

```python
def is_safe_url(url, allowed_hosts, require_https=False):
    """
    Return ``True`` if the url is a safe redirection (i.e. it doesn't point to
    a different host and uses a safe scheme).

    Always return ``False`` on an empty url.

    If ``require_https`` is ``True``, only 'https' will be considered a valid
    scheme, as opposed to 'http' and 'https' with the default, ``False``.
    """
```


[^1]: https://docs.djangoproject.com/en/2.1/releases/2.1/#miscellaneous
[^2]: https://docs.djangoproject.com/en/2.1/_modules/django/utils/http/